sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -245,6 +245,10 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
 
     (1 to 100).map(i => (i, i.toString)).toDF("key", "value").createOrReplaceTempView("testdata")
 
+    Seq((1, 1), (1, 2), (2, 1), (2, 2), (3, 1), (3, 2))
```

Comment:
To be honest, it is hard to write test data, especially when we want very few rows in each data set.